src/test/java/mate/academy/validation/EmailValidatorTest.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.validation;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class EmailValidatorTest {
+    private EmailValidator emailValidator;
+
+    @BeforeEach
+    void setUp() {
+        emailValidator = new EmailValidator();
+    }
+
+    @Test
+    void isValid_True() {
```

Comment:
I think it's better to mock ConstraintValidatorContext

Reply:
test added